docs/networks/arbitrum-goerli.md

```diff
@@ -0,0 +1,109 @@
+| graph-node      | [0.31.0-rc.0](https://github.com/graphprotocol/graph-node/releases/tag/v0.31.0-rc.0) |
+
+## Network Parameters
+
+| Parameter                   | Value             |
+| --------------------------- | ----------------- |
+| Epoch length                | ~ 2h (554 blocks) |
+| Maximum allocation lifetime | ~ 9h (4 epochs)   |
+
+## Contracts & accounts
+
+| Name               | Address                                                                                                                        |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------ |
+| Graph Token (GRT ) | [`0x18C924BD5E8b83b47EFaDD632b7178E2Fd36073D`](https://goerli.arbiscan.io/address/0x18C924BD5E8b83b47EFaDD632b7178E2Fd36073D) |
+| Staking            | [`0xcd549d0C43d915aEB21d3a331dEaB9B7aF186D26`](https://goerli.arbiscan.io/address/0xcd549d0C43d915aEB21d3a331dEaB9B7aF186D26) |
+| Data Edge          | [``](https://goerli.arbiscan.io/address/) |
+| Block Oracle Owner | [``](https://goerli.arbiscan.io/address/) |
+
+Other network contracts can be found in [graphprotocol/contracts](https://github.com/graphprotocol/contracts/blob/dev/addresses.json#L971).
+
+## Configuration
+
+The Graph testnet contracts live on Goerli, but many of the subgraphs used in the
+testnet (for now) are Mainnet subgraphs. This means:
+
+- Indexer Agent and Indexer Service must connect to Goerli
+- Graph Node must connect to at least one Mainnet Ethereum node/provider
+
+### Indexer Agent
+
+| Environment Variable                        | CLI Argument                    | Value                                                                                       |
+| ------------------------------------------- | ------------------------------- | ------------------------------------------------------------------------------------------- |
+| `INDEXER_AGENT_ETHEREUM`                    | `--ethereum`                    | An Ethereum Goerli node/provider                                                            |
+| `INDEXER_AGENT_ETHEREUM_NETWORK`            | `--ethereum-network`            | `goerli`                                                                                    |
+| `INDEXER_AGENT_INDEXER_ADDRESS`             | `--indexer-address`             | Ethereum address of testnet indexer                                                         |
+| `INDEXER_AGENT_INDEXER_GEO_COORDINATES`     | `--indexer-geo-coordinates`     | Geo coordinates of testnet indexer infrastructure                                           |
+| `INDEXER_AGENT_MNEMONIC`                    | `--mnemonic`                    | Ethereum mnemonic for testnet operator                                                      |
+| `INDEXER_AGENT_NETWORK_SUBGRAPH_DEPLOYMENT` | `--network-subgraph-deployment` | `QmPVz18RFwK6hE5rZFWERk23LgrTBz2FCkZzgPSrFxFWN4`                                            |
```

Comment:
I believe this is the `Goerli` network subgraph deployment id, not `arbitrum-goerli`.  

The IPFS supported subgraph for Arbitrum Goerli is `Qmd7zHW36Ep6wDJCBgw6DnvKV3Brzq8naPUDVnZqSV2Wwj` -  https://thegraph.com/hosted-service/subgraph/graphprotocol/graph-network-arbitrum-goerli. 
We should find the non-ipfs version though because that's all that is required for indexers.